rust/arrow/Cargo.toml

```diff
@@ -48,11 +49,12 @@ lazy_static = "1.4"
 packed_simd = { version = "0.3.4", optional = true, package = "packed_simd_2" }
 chrono = "0.4"
 flatbuffers = "0.6"
+bitvec = "0.19"
 hex = "0.4"
 prettytable-rs = { version = "0.8.0", optional = true }
 
 [features]
-default = []
+default = ["simd"]
```

Comment:
we probably shouldn't change the default to include simd, as we'd like the default features to allow users to compile with stable.

Reply:
Oh forgot to set it back.